db/sql/400_api.sql

```diff
@@ -91,4 +93,16 @@ RETURNS setof json as $$
   SELECT cast(ST_AsGeoJSON(sc.*) AS json)
   FROM selected_cols AS sc;
 $$ LANGUAGE SQL VOLATILE;
```

Comment:
Not touched in this PR, but I think `STABLE` can be used in this function for potentially better query optimization than with stricter `VOLATILE`. `STABLE` allows for database lookups (which `IMMUTABLE` does not) that we do here, this is just a "read-only" function whose result depends on the argument values and the database state. `VOLATILE` would be needed if we modified the database with the function and/or used values from functions like `random()`.

You can try changing this within this PR or just add a TODO.